packages/salesforcedx-vscode-apex-debugger/README.md

```diff
@@ -1,3 +1,146 @@
 # salesforcedx-vscode-apex-debugger
-This extension enables VS Code to use the Apex Debugger with your scratch orgs.
-It requires at least SFDX CLI v41.1.0 and sandbox instance v41.0.
\ No newline at end of file
+This extension enables VS Code to use the real-time Apex Debugger with your scratch orgs.
+
+## Prerequisites
+Before you set up the Apex Debugger, make sure that you have these essentials.
+
+* At least one available Apex Debugger session in your Dev Hub org
+    * One Apex Debugger session is included with Performance Edition orgs, Unlimited Edition orgs, and License Management Orgs.
+    * To purchase Apex Debugger sessions for Enterprise Edition orgs, or to purchase more sessions for orgs that already have allocated sessions, contact Salesforce.
+* [Salesforce CLI](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm) v41.1.0 or later
+* The latest versions the [salesforcedx-vscode-core](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-core) and [salesforcedx-vscode-apex-debugger](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-apex-debugger) extensions (we suggest that you install all extensions in the [salesforcedx-vscode](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode) extension pack)
+
+## Set Up the Apex Debugger
+The first time that you use the Apex Debugger in VS Code, complete these setup steps.
+
+1. Add the `DebugApex` feature to a scratch org definition file:  
+  `"features": "DebugApex"`
+1. In VS Code, run **SFDX: Create a Default Scratch Org**. 
+1. Choose the scratch org definition file that includes the `DebugApex` feature.
+1. When your scratch org is ready, assign permissions for the Apex Debugger to the org’s admin user.  
+    1. In VS Code, run **SFDX: Open Default Scratch Org**.
+    1. In your browser, from Setup, enter `Permission Sets` in the Quick Find box, then select **Permission Sets**.
+    1. Create a permission set.
+    1. Give the permission set a name that you can remember, such as `Debug Apex`.
+    1. In the “Select the type of users who will use this permission set” section, choose **None** from the User License drop-down list. Choosing None lets you assign the permission set to more than one type of user.
+    1. Save your changes.
+    1. Click **System Permissions**.
+    1. Click **Edit**.
+    1. Enable **Debug Apex**. Enabling this permission enables the set of permissions that your user needs. 
```

Comment:
nit: not sure explanatory sentence adds a ton of new info - I would consider removing it